### PR TITLE
Updated Footer Page

### PR DESCRIPTION
**Pull request by Tushar00002**
Screenshot (413)
Screenshot (414)
Screenshot (418)
Screenshot (419)
Screenshot (420)

### CODE REVIEW

File: client/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                      </div>
+                      <div className="bg-[#CAD2C5]  w-12 h-12 rounded-full flex items-center justify-center mr-4 shadow-2xl">
+                        <BsTwitter className=" text-[#2F3E46] shadow-2xl" />
+                      </div>
+                      <div className="bg-[#CAD2C5]  w-12 h-12 rounded-full flex items-center justify-center mr-4 shadow-2xl">
+                        <BsLinkedin className=" text-[#2F3E46] shadow-2xl" />
+                      </div>
+                      <div className="bg-[#CAD2C5]  w-12 h-12 rounded-full flex items-center justify-center mr-4 shadow-2xl">
+                        <BsYoutube className=" text-[#2F3E46]" />
+                      </div>
```
Comment (vishwa511) [inline client/src/components/Footer/Footer.jsx:102]: Good work, just made a small change, as you centrally aligned the row of events details and row of social updates, as per figma design align them left👍
s3

**Review by vishwa511 — CHANGES_REQUESTED** [review]

File: client/src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          {/* sdv */}
+
```
Comment (sanskaromar) [inline client/src/components/Footer/Footer.jsx:111]: Unclear comment

**Review by sanskaromar — CHANGES_REQUESTED** [review]

**Review by NeerajChatterjee — CHANGES_REQUESTED** [review]
image
You can see, there are two up arrows.  But, we need only one up arrow that will be placed at the bottom for the 'scroll-to-top' feature. 

image
In the mobile design, make the bottom container w-screen and don't leave the left side space blank as it squeezes the social logos. Desktop view is perfect for the bottom container.

File: client/src/components/Footer/Footer.jsx (hunk 1 above)
Comment (Tushar00002) [reply client/src/components/Footer/Footer.jsx:98]: aligned the text to left

File: client/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  BsYoutube,
+} from "react-icons/bs";
+import { RiArrowDropUpLine } from "react-icons/ri";
+const Footer = () => {
+  return (
+    <>
+      <div className="Page  w-full bg-[#1b1c1c] pt-8">
+        <div className="Header text-white text-[48px] lg:text-[52px] flex flex-col justify-center items-center p-4">
+          <p className="">We Have Top Executives</p>
+          <p className="">And</p>
```
[the remark this answers is not in the document]
Comment (Tushar00002) [reply client/src/components/Footer/Footer.jsx]: Removed empty classnames

File: client/src/components/Footer/Footer.jsx (hunk 2 above)
Comment (Tushar00002) [reply client/src/components/Footer/Footer.jsx]: Removed comments

**Comment by Tushar00002** [comment]
Made the required changes

**Review by sanskaromar — APPROVED** [review]

**Review by sanskaromar — APPROVED** [review]
Try to write proper commit messages.

**Pull request merged**